addons/product/models/product_attribute.py

```diff
@@ -117,10 +117,11 @@ class ProductTemplateAttributeLine(models.Model):
     _name = "product.template.attribute.line"
     _rec_name = 'attribute_id'
     _description = 'Product Template Attribute Line'
-    _order = 'attribute_id, id'
+    _order = 'sequence, id'
```

Comment:
I would probably keep attribute_id after sequence to keep the existing order when they all have the same sequence, which is going to be the case for all existing lines.